src/assets/lang/en.json

```diff
@@ -1,6 +1,10 @@
 {
-  "header": "OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies.",
-  "stats": "{variablesCount, number} variables and {parametersCount, number} parameters referenced to modelize the french tax-benefit system.",
+  "header":"OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies.",
+  "learnMore":" Learn more.",
```

Comment:
Why is there a leading space inside of the message? This usually makes it harder to compose messages when they include leading or trailing space.
My previous comment on a missing space was after the JSON colon 🙂